third_party/nvfuser/csrc/python_frontend/fusion_definition.h

```diff
@@ -3,18 +3,20 @@
 
 #include <c10/macros/Export.h>
 #include <kernel_cache.h>
+#include <python_frontend/fusion_state.h>
 
 namespace nvfuser::python_frontend {
 
 class FusionCache;
 class FusionInterface;
+class FusionState;
 struct RecordFunctor;
 struct UserSchedule;
 
 //! This is helper function used to print a python formated
 //! Fusion IR DataType when printing a fusion definition.
 
-TORCH_CUDA_CU_API const char* dtypeToPyString(nvfuser::PrimDataType t);
+TORCH_CUDA_CU_API const char* dtypeToPyString(PrimDataType t);
 
 //! The State and the StateType enum are used to define state objects to
 //! encapsulate the recording of state in the FusionDefinition.
```

Comment:
I would also put the `StateType` enum and the `State` struct with in your `fusion_state.h` file given I don't believe they are needed by the `FusionDefinition` anymore, correct?